testing/trino-testing-services/src/main/java/io/trino/testing/services/junit/LogTestDurationListener.java

```diff
@@ -193,10 +190,7 @@ public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult
         }
 
         try {
-            Duration duration = endTest(testIdentifier);
-            if (duration.compareTo(TEST_LOGGING_THRESHOLD) > 0) {
-                log.info("Test %s took %s", testName(testIdentifier), duration);
```

Comment:
A process question. This code was recently added (in https://github.com/trinodb/trino/pull/20539).
Why people involved in that PR were not asked for input or CCd before this "partial revert" PR was merged?